lib/helper.js

```diff
@@ -3,15 +3,14 @@ var React = require('react/addons');
 var TestUtils = React.addons.TestUtils;
 
 /**
- * Renders a component
+ * Renders a component. Method signature is the same as
+ * https://facebook.github.io/react/docs/top-level-api.html#react.createelement
  *
  * @method renderComponent
- * @param {ReactComponent} Component class
- * @param {Object} props used to initialize the component
  * @return {ReactElement} rendered react element
  **/
-function renderComponent(Component, props) {
-    var element = React.createElement(Component, props);
+function renderComponent() {
+    var element = React.createElement.apply(React, arguments);
```

Comment:
it was already possible to pass the childdren in props but i like this feature coz it matches the createElement, can you just add the extra argument to the function instead of using apply?

```js
function renderComponent(Component, props, children) {
   var element = React.createElement(Component, props, children);
   ...
}
```